### PR TITLE
update make

### PR DESCRIPTION
**Pull request by jordansread**

### CODE REVIEW

**Comment by jordansread** [comment]
@lawinslow take a look?

File: scripts/lake_summary_nldas.R [hunk 1]
```diff
@@ -151,33 +151,41 @@ lake_driver_nldas <- function(file='data/NLDAS_data/NLDAS_driver_file_list.tsv')
...
-                }, error = function(e){
-                  cat('rsync of ', file, ' FAILED **', file=mssg.file, append = TRUE)
-                })
-              } 
+                save(data.site, file=local.file, compress="xz")
+                
+              }
+              tryCatch({
+                message('begining rsync ', Sys.time())
+<<<<<<< Updated upstream
```
Comment (lawinslow) [inline scripts/lake_summary_nldas.R]: this need to be cleaned up?

Comment (jordansread) [reply scripts/lake_summary_nldas.R]: ugg, yeah. good catch

Comment (jordansread) [reply scripts/lake_summary_nldas.R]: missed that file I guess.

**Comment by jordansread** [comment]
@lawinslow I'd like to merge and tag this before I start working on the data release metadata for the tri-state files. I am modifying the metadata template and would like to be able to come back to this version if we need to, but want to move on.

File: configs/nldas_config.yml [hunk 2]
```diff
@@ -5,8 +5,8 @@ nldas_url: "dods://hydro1.sci.gsfc.nasa.gov/dods/NLDAS_FORA0125_H.002"
 data_times: ["1979-01-04","2016-01-01"]
 data_variables: ['spfh2m','tmp2m','pressfc','apcpsfc','dlwrfsfc','ugrd10m','vgrd10m','dswrfsfc']
 variable_names: ['spfh2m','tmp2m','pressfc','apcpsfc','dlwrfsfc','ugrd10m','vgrd10m','dswrfsfc']
-wps_url: "http://cida.usgs.gov/gdp/process/WebProcessingService"
-data_url: "dods://cida-eros-netcdfdev.er.usgs.gov:8080/thredds/dodsC/thredds_workspace/nldas_miwimn/nldas_miwimn.ncml"
+wps_url: "http://cida-test.er.usgs.gov/gdp/process/WebProcessingService"
```
Comment (lawinslow) [inline configs/nldas_config.yml:8]: should this be test or prod?

**Comment by lawinslow** [comment]
looks good

File: configs/nldas_config.yml (hunk 2 above)
Comment (jordansread) [reply configs/nldas_config.yml:8]: :giggle: I switch between the two based on which one is acting up :(

**Pull request merged**